src/app/child-dev-project/schools/school-select/school-select.component.html

```diff
@@ -3,7 +3,7 @@
     <input
       #inputField
       matInput
-      placeholder="Schools"
+      placeholder="Teams"
```

Comment:
I dont think this should be hardcoded. Maybe use the entity type?

Reply:
yes, waiting for PR #728 for a proper solution. I am reverting these - this was just a workaround to build a beta version for the next customer appointment